PurchasesCoreSwift/SubscriberAttributes/AttributionDataMigrator.swift

```diff
@@ -0,0 +1,145 @@
+//
+//  AttributionDataMigrator.swift
+//  PurchasesCoreSwift
+//
+//  Created by César de la Vega on 6/16/21.
+//  Copyright © 2021 Purchases. All rights reserved.
+//
+
+import Foundation
+
+@objc(RCAttributionDataMigrator) public class AttributionDataMigrator: NSObject {
+
+    @objc public func convertAttributionDataToSubscriberAttributes(
+            attributionData: [String: Any], network: Int
+    ) -> [String: Any] {
+        let network = AttributionNetwork(rawValue: network)
+        let attributionData = attributionData.filter { !($0.1 is NSNull) }
```

Comment:
let's perhaps be a bit more explicit in calling this $0.value at least, so it doesn't look cryptic i.e.: $0.1

Reply:
or maybe even make an extension: 
i.e.: 
```swift
extension Dictionary {
    func removingNSNullValues() -> Dictionary {
        self.filter { !($0.1 is NSNull) }
   }
}

...

let attributionData = attributionData.removingNSNullValues()
```